## First hint of strangeness

- Friedrich Wilhelm Bessel discovered irregularities in the path of [Sirius](https://earthsky.org/constellations/canis-major-the-greater-dog-sirius/), the brightest star in the sky. 
- He suspected that Sirius is a part of a binary system but could not spot its companion.
- The parallax of Sirius was measured to be $0.38''$. 
- It allows us to estimate the distance from the sun.

In [7]:
import numpy as np
import scipy.constants as sc

parallax_s = 0.38 # in seconds
parallax_d = parallax_s/3600 # in degrees
parallax_r = parallax_d * np.pi/180 # in radians

distance_au = 1/parallax_r
print(f'Sirius is at a distance of {distance_au:.0f} AU from the sun.')

Sirius is at a distance of 542802 AU from the sun.


## A companion is found

- Alvan Clark Jr accidentally discovered a faint companion to Sirius when he was testing a new lens produced by his father’s company.
- Absolute visual magnitude of the companion, now called Sirius B but Eddington called is Sirius _coms_, is $11.18$.
- The original Sirius, now called Sirius A, has an absolute visual magnitude of $1.43$. This allows us to compare their brightness using the formula
$$
m_1 - m_2 = \frac{5}{2}\log_{10}\left(\frac{F_1}{F_2}\right)
$$

In [8]:
mA = 1.43
mB = 11.18
x = 1/10**(2 * (mA - mB)/5)
print(f'Sirius A is {x:0.2f} times brighter than Sirius B.')

Sirius A is 7943.28 times brighter than Sirius B.


## Comparison with the sun

- Let us compare the brightness of these stars with that of the sun.

In [9]:
mS = 4.83
xA = 1/10**(2 * (mA - mS)/5)
xB = 10**(2 * (mB - mS)/5)
print(f'Sirius A is {xA:0.2f} times brighter than the sun.')
print(f'The sun is is {xB:0.2f} times brighter than Sirius B.')

Sirius A is 22.91 times brighter than the sun.
The sun is is 346.74 times brighter than Sirius B.


## The orbit of the binary system

- The period of Sirius A (and B) around the centre of mass of the binary system was measured to be about $50$ years.
- The semi-major axis of the elliptical orbit was found to be roughly $7.5''$. Therefore, its length is

In [10]:
theta_d = 7.5/3600
theta_r = theta_d * np.pi/180
a = distance_au * theta_r
print(f'The semi-major axis of the binary system is {a:0.2f} AU.')

The semi-major axis of the binary system is 19.74 AU.


- This information suffices to calculate the mass of the binary system.

## In keeping with LAAG tradition

- Kepler's third law for a binary system is
$$
T^2 = \frac{4\pi^2}{G(m_1 + m_2)}a^3.
$$
- We can calculate $m_1 + m_2$ using this equation.

In [11]:
AU = 149597870700 # One astronomical unit in m.
SO = 1.989e30 # One solar mass in kg.

a_m = a * AU # Semi-major axis in metres.
T_s = 50 * 3.154e7 # Period in seconds.
sum_of_masses_kg = 4 * np.pi**2 * a_m**3/(sc.G * T_s**2)
sum_of_masses_sm = sum_of_masses_kg/SO
print(f'Mass of the Sirius binary system is {sum_of_masses_sm:0.2f} solar masses.')

Mass of the Sirius binary system is 3.08 solar masses.


- Otto Struve measured the angular separation between Sirius A and B to be thrice the separation between the centre of mass and Sirius A.
- Sirius A is therefore twice as massive as Sirius B.

## A different kind of a star
- We now use the mass-luminosity relation to calculate the mass of the two stars. For a main-sequence star, it is
$$
\frac{L}{L_\odot} = \left(\frac{M}{M_\odot}\right)^{4}.
$$
for stars of masses between $0.43$ and $2\;M\odot$.
- We calculated the ratio on the left hand side to be $22.91$. Therefore,

In [12]:
print(f'The mass of Sirius A is {xA**(1/4):0.2f} times that of the sun.')
print(f'Therefore mass of Sirius B is {sum_of_masses_sm - xA**(1/4):0.2f} times that of the sun.')

The mass of Sirius A is 2.19 times that of the sun.
Therefore mass of Sirius B is 0.89 times that of the sun.


- These are crude calculations. More accurate ones give the mass of Sirius B to be roughly $1\;M_\odot$ and that of Sirius A to to $2\;M_\odot$.
- Why is a star as massive as the sun almost $350$ times dimmer?

## The curious case of 40 Eridani B

- [40 Eridani](https://en.wikipedia.org/wiki/40_Eridani#Properties) is a system of triple stars.
- Of these, the prominent member is of spectral type K, a red dwarf.
- Williamina Flemming classified 40 Eridani B in spectral class A - a much hotter star.
- The mass of 40 Eridani B was found to be approximately 45% of that of the sun (spectral class G). It is much smaller than the sun and yet is much hotter.
- Around the same time, people studied the spectrum of Sirius B and found it to be composed mostly of hydrogen lines, that is spectral class A.

## How big are these stars?

- Stepfan-Boltzman law states that radiant emittance (radiant power per unit area) is proportional to the fourth power of absolute temperature. The constant of proportionality is called the _Stefan-Boltzmann constant_, $\sigma$.
- A sphere of radius $r$ thus emits $4\pi r^2 \sigma T^4$. But this is just the luminosity.
- We calculated that the luminosity of Sirius A is $7943.28$ times more than that of Sirius B. But they belong to the same spectral class, that is, they have the same temperature. If $L_A$ and $L_B$ are the luminosities of Sirius A and B, $r_A$ and $r_B$ their radii, $T_A$ and $T_B$ their temperatures then
$$
\begin{eqnarray}
L_A &=& 4\pi r_A^2 \sigma T_A^4 \\
L_B &=& 4\pi r_B^2 \sigma T_B^4.
\end{eqnarray}
$$
If $T_A = T_B$ and $L_A = 7943.28 L_B$ then 
$$
\frac{r_A}{r_B} = \sqrt{\frac{L_A}{L_B}} \approx 89.
$$
Thus, Sirius B was a hundred times tinier than Sirius B, although both were equally hot.

- William Luyten called such stars _white dwarfs_. He discovered a large number of them in his seven decades long career.

## Quantum stars

- At the temperature of a white dwarf matter is fully ionised.
- If there is no more energy production, the matter will cool and will return to state of neutral atoms. 
- But this means that the density of the matter must drop by $5$ orders of magnitude, that is, the star's volume will have to rise by $5$ orders of magnitude.
- If the star must expand, it must do work against gravity. Who will supply the energy for this endeavour?
- The star continues to radiate and lose energy and yet cannot expand. It is "a body continually losing heat but having insufficient energy to grow cold", complained Eddington.
- Eddington's concern cannot be addressed by classical physics.
- R. H. Fowler resolved the paradox within three months of the paper announcing _Fermi-Dirac Statistics_.